app/models/user.rb

```diff
@@ -108,15 +108,13 @@ def accessible_vita_partners
     when AdminRole::TYPE
       VitaPartner.all
     when OrganizationLeadRole::TYPE
-      VitaPartner.organizations.where(id: role.organization).or(
-        VitaPartner.sites.where(parent_organization_id: role.organization)
-      )
+      VitaPartner.organizations.where(id: role.organization) + VitaPartner.sites.where(parent_organization_id: role.organization)
```

Comment:
The difference is that `+` will (I believe) eagerly execute both queries.

Here's why I think that:

```
irb(main):004:0> x = VitaPartner.all; nil
=> nil
irb(main):005:0> x = (VitaPartner.all + VitaPartner.all); nil
  VitaPartner Load (1.3ms)  SELECT "vita_partners"."id", "vita_partners"."allows_greeters", "vita_partners"."archived", "vita_partners"."capacity_limit", "vita_partners"."coalition_id", "vita_partners"."created_at", "vita_partners"."logo_path", "vita_partners"."name", "vita_partners"."national_overflow_location", "vita_partners"."parent_organization_id", "vita_partners"."processes_ctc", "vita_partners"."timezone", "vita_partners"."updated_at" FROM "vita_partners" ORDER BY "vita_partners"."name" ASC
  VitaPartner Load (0.7ms)  SELECT "vita_partners"."id", "vita_partners"."allows_greeters", "vita_partners"."archived", "vita_partners"."capacity_limit", "vita_partners"."coalition_id", "vita_partners"."created_at", "vita_partners"."logo_path", "vita_partners"."name", "vita_partners"."national_overflow_location", "vita_partners"."parent_organization_id", "vita_partners"."processes_ctc", "vita_partners"."timezone", "vita_partners"."updated_at" FROM "vita_partners" WHERE "vita_partners"."parent_organization_id" IN ($1, $2, $3, $4, $5, $6) ORDER BY "vita_partners"."name" ASC, "vita_partners"."id" ASC  [["parent_organization_id", 1], ["parent_organization_id", 2], ["parent_organization_id", 3], ["parent_organization_id", 6], ["parent_organization_id", 5], ["parent_organization_id", 4]]
  VitaPartner Load (0.6ms)  SELECT "vita_partners"."id", "vita_partners"."allows_greeters", "vita_partners"."archived", "vita_partners"."capacity_limit", "vita_partners"."coalition_id", "vita_partners"."created_at", "vita_partners"."logo_path", "vita_partners"."name", "vita_partners"."national_overflow_location", "vita_partners"."parent_organization_id", "vita_partners"."processes_ctc", "vita_partners"."timezone", "vita_partners"."updated_at" FROM "vita_partners" WHERE "vita_partners"."parent_organization_id" IN ($1, $2) ORDER BY "vita_partners"."name" ASC, "vita_partners"."id" ASC  [["parent_organization_id", 3], ["parent_organization_id", 6]]
  VitaPartner Load (0.6ms)  SELECT "vita_partners"."id", "vita_partners"."allows_greeters", "vita_partners"."archived", "vita_partners"."capacity_limit", "vita_partners"."coalition_id", "vita_partners"."created_at", "vita_partners"."logo_path", "vita_partners"."name", "vita_partners"."national_overflow_location", "vita_partners"."parent_organization_id", "vita_partners"."processes_ctc", "vita_partners"."timezone", "vita_partners"."updated_at" FROM "vita_partners" ORDER BY "vita_partners"."name" ASC
  VitaPartner Load (0.5ms)  SELECT "vita_partners"."id", "vita_partners"."allows_greeters", "vita_partners"."archived", "vita_partners"."capacity_limit", "vita_partners"."coalition_id", "vita_partners"."created_at", "vita_partners"."logo_path", "vita_partners"."name", "vita_partners"."national_overflow_location", "vita_partners"."parent_organization_id", "vita_partners"."processes_ctc", "vita_partners"."timezone", "vita_partners"."updated_at" FROM "vita_partners" WHERE "vita_partners"."parent_organization_id" IN ($1, $2, $3, $4, $5, $6) ORDER BY "vita_partners"."name" ASC, "vita_partners"."id" ASC  [["parent_organization_id", 1], ["parent_organization_id", 2], ["parent_organization_id", 3], ["parent_organization_id", 6], ["parent_organization_id", 5], ["parent_organization_id", 4]]
  VitaPartner Load (0.5ms)  SELECT "vita_partners"."id", "vita_partners"."allows_greeters", "vita_partners"."archived", "vita_partners"."capacity_limit", "vita_partners"."coalition_id", "vita_partners"."created_at", "vita_partners"."logo_path", "vita_partners"."name", "vita_partners"."national_overflow_location", "vita_partners"."parent_organization_id", "vita_partners"."processes_ctc", "vita_partners"."timezone", "vita_partners"."updated_at" FROM "vita_partners" WHERE "vita_partners"."parent_organization_id" IN ($1, $2) ORDER BY "vita_partners"."name" ASC, "vita_partners"."id" ASC  [["parent_organization_id", 3], ["parent_organization_id", 6]]
=> nil
```

the `; nil` trick is a way to avoid any SQL executions from the REPL printing the expression value.

I'd like to see this fixed if possible; for one thing, it makes the result type an `Array` not a `Relation`, which may cause problems with which methods it has.